src/app/services/scheduler/scheduler.service.ts

```diff
@@ -82,8 +82,10 @@ export class SchedulerService {
 
   fetchAppList(partitionName: string, queueName: string): Observable<AppInfo[]> {
     const appsUrl = `${this.envConfig.getSchedulerWebAddress()}/ws/v1/partition/${partitionName}/queue/${queueName}/applications`;
+    const headers = new HttpHeaders();
+    headers.set('Accept-Encoding', 'gzip')
```

Comment:
Does it work with scheduler without https://github.com/apache/yunikorn-core/pull/757 ?

Reply:
it should just be ignored if the server does not support it. Fallback will always be identity transform (i.e. no compression)